stylesheets/abstracts/_variables.scss

```diff
@@ -2,74 +2,39 @@
 // This file contains all application-wide Sass variables.
 // -----------------------------------------------------------------------------
 
-// Colors
-// Color names derived from https://www.color-blindness.com/color-name-hue/
+// Colors by category
 
-//TODO: Consolidate colors by moving to a single hue (for blues and greys) and use consistent Sass color function math to create variations in components.
-
-// Crusta Orange
-$crusta-orange: hsl(13, 86%, 60%) !default;
-
-// Flame Orange
+// Oranges
 $flame-orange: hsl(17, 76%, 48%) !default;
-
-// Umber Brown
-$umber-brown: hsl(17, 87%, 38%) !default;
-
-// Rust Red
-$rust-red: hsl(19, 83%, 34%) !default;
-
-// Maroon Brown
-$maroon-brown: hsl(23, 100%, 20%) !default;
-
-// Sandy Orange
+$crusta-orange: hsl(13, 86%, 60%) !default;
 $sandy-orange: hsl(25, 87%, 64%) !default;
-
-// Peach Orange
 $peach-orange: hsl(30, 100%, 80%) !default;
+$rose-violet: hsl(0, 100%, 95%) !default;
 
-// Wan White Grey
-$wan-white-grey: hsl(40, 10%, 88%) !default;
-
-// Silver Grey
-$silver-grey: hsl(45, 6%, 72%) !default;
-
-// Desert Grey
-$desert-grey: hsl(60, 10%, 96%) !default;
-
-// Water Blue
+// Blues
+$regal-blue: hsl(214, 49%, 19%) !default;
+$midnight-blue: hsl(214, 72%, 24%) !default;
+$yale-blue: hsl(214, 70%, 35%);
```

Comment:
Is there a reason this color doesn't have a `!default`?

Reply:
Good catch! Just an oversight.